code/modules/mob/living/brain/brain_item.dm

```diff
@@ -79,8 +79,8 @@
 		if(!brainmob.stored_dna)
 			brainmob.stored_dna = new /datum/dna/stored(brainmob)
 		C.dna.copy_dna(brainmob.stored_dna)
-		if(L.disabilities & NOCLONE)
-			brainmob.disabilities |= NOCLONE	//This is so you can't just decapitate a husked guy and clone them without needing to get a new body
+		if(L.has_disability(NOCLONE))
+			brainmob.disabilities[NOCLONE] = L.disabilities[NOCLONE]
```

Comment:
Why is this not using the helper ?

Reply:
this is copying all the sources from the mob to the brain; i could make a separate helper for it, if you want me to

Reply:
No, that's fine as is.